simple_history/models.py

```diff
@@ -180,9 +180,9 @@ def contribute_to_class(self, cls, name):
     def add_extra_methods(self, cls):
         def save_without_historical_record(self, *args, **kwargs):
             """
-            Save model without saving a historical record
+            Save the model instance without creating a historical record.
 
-            Make sure you know what you're doing before you use this method.
+            *Make sure you know what you're doing before using this method.*
```

Comment:
nit: I'm not a fan of trying to use markup here.

Reply:
We do have somewhat similar markup in other docstrings (like double instead of single backticks) that's useful when rendering them with previewing tools in some IDEs 🤷 Though I do see that it doesn't add that much value in this case, and it looks slightly out of place when reading the code directly, so I'll remove it 🙂